.github/workflows/python-lint.yml

```diff
@@ -0,0 +1,32 @@
+# This GitHub Actions workflow is responsible for linting Python code in the repository.
+# It is triggered on every 'push' and 'pull_request' event. The workflow runs on an
+# Ubuntu-latest VM and uses the Black code formatter to check the style of the code
+# located in the "./kay" directory.
+
+name: Python Lint
+
+on: [ push, pull_request ]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [ '3.8', '3.9', '3.10', '3.11' ]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v3
+        with:
+          python-version: ${{ matrix.python-version }}
```

Comment:
The linter will be run 4 times for each python version in the matrix and it will be run on the GitHub side. It makes sure that the formatting works for all those versions. Sure, let's use 3.11.